src/test/java/org/sagebionetworks/bridge/exporter/integration/ExportTest.java

```diff
@@ -0,0 +1,1142 @@
+            List<String> studyStatusRowContent = studyStatusTableContents.get(offset).getValues();
+
+            for (int i = 0; i < studyStatusTableHeaders.size(); i++) {
+                String headerName = studyStatusTableHeaders.get(i).getName();
+                String columnValue = studyStatusRowContent.get(i);
+
+                switch (headerName) {
+                    case "uploadDate": {
+                        assertEquals(uploadStatus.getRecord().getUploadDate().toString(), columnValue);
+                        break;
+                    }
+                    default: fail("not expected column type: " + headerName + ": " + columnValue);
+                }
+            }
+
+            QueryResultBundle queryResultSurvey = null;
+            List<Row> surveyTableContents = null;
+
+            for (int i = 0; i < SYNAPSE_QUERY_RETRIES; i ++) {
+                LOG.info("Re-query synapse survey table times: " + i);
+                for (int j = 0; j < SYNAPSE_RETRIES; j++) {
+                    try {
+                        LOG.info("Retry get synapse survey table query result times: " + j);
+                        queryResultSurvey = synapseClient.queryTableEntityBundleAsyncGet(jobIdTokenSurvey, schemaKeyTableId);
+                        break;
+                    } catch (SynapseResultNotReadyException e) {
+                        TimeUnit.SECONDS.sleep(SYNAPSE_SECONDS);
+                    }
+                }
+
+                if (queryResultSurvey == null) {
+                    fail("Query survey table failed.");
+                }
+
+                surveyTableContents = queryResultSurvey.getQueryResult().getQueryResults().getRows();
+                if (surveyTableContents.size() != 0) {
+                    break;
+                }
+
+                TimeUnit.SECONDS.sleep(SYNAPSE_RETRY_DELAY);
+
+                jobIdTokenSurvey = synapseClient.queryTableEntityBundleAsyncStart("select * from " + schemaKeyTableId
+                                + " where recordId=\'" + uploadStatus.getRecord().getId() + "\'"
+                        , 0L, 100L, true, 0xF, schemaKeyTableId);
+            }
+
+            // verify the size of the table is equal to the number of uploads
+            assertEquals(1, surveyTableContents.size());
+
+            List<SelectColumn> surveyTableHeaders = queryResultSurvey.getQueryResult().getQueryResults().getHeaders();
+            List<String> surveyRowContent = surveyTableContents.get(0).getValues();
+
+            // verify each column
+            for (int i = 0; i < surveyTableHeaders.size(); i++) {
+                String headerName = surveyTableHeaders.get(i).getName();
+                String columnValue = surveyRowContent.get(i);
+
+                switch (headerName) {
+                    case "recordId": {
+                        assertEquals(uploadStatus.getRecord().getId(), columnValue);
+                        break;
+                    }
```

Comment:
There's a lot of duplicate code between here and the appVersion table. Can we consolidate?